RELEASE-NOTES.md

```diff
@@ -6,6 +6,104 @@ OpenDungeons since version 0.4.7, in reverse chronological order.
 The numbers preceded by a hash (e.g. #9) reference GitHub issue numbers on
 our repository at https://github.com/OpenDungeons/OpenDungeons
 
+### Version 0.7.0
+
+**Highlights:**
+
+* Multiplayer lobby using a master server, to find other players all over the world
+* Animated main menu with scripted events!
+* Play voice-overs on some game events
+* Vastly improved creatures behaviour, including support for spell casting,
+  elemental damage, better ranged behaviour, and many customisation features
+* New rooms: Arena, Bridges, Casino, Prison, Torture room
+* New creatures: Cultist, Elf, Dark Elf, Lich, Nature Monster, Rat, Roach, Skeleton
```

Comment:
Ah right, I added them in the graphics section but not in the highlights. Will add them when putting the release data, or if other stuff has to be added before.